### ISSUE TITLE
Where to start?

### ISSUE THREAD

**Issue by wonganikalouer** [wonganikalouer/chithunzi.com#4]
Hello devs, where exactly should we start developments, the System Dashboard or the Main Website?
#I prefer going with the Main Website with dummy data.

**Comment by jmziya** [comment]
Yeah the main website obviously with dummy data. I second. Create an for dashboard though. Others might prefer starting with that.

**Comment by wonganikalouer** [comment]
ok, ill start designing the UI,

**Comment by jmziya** [comment]
Okay Great. On "create an dashboard", I meant create an issue for dashboard. You should create issues for various tasks. Make them tiny tasks, so contributors can be contributing base on those
